netdata-installer.sh

```diff
@@ -550,9 +557,16 @@ bundle_libmosquitto
 # -----------------------------------------------------------------------------
 
 build_libwebsockets() {
+  CFLAGS=
+  LDFLAGS=
+  CXXFLAGS=
+
   pushd "${1}" > /dev/null || exit 1
   if [ "$(uname)" = "Darwin" ] && [ -d /usr/local/opt/openssl ]; then
-    run env CFLAGS= CXXFLAGS= LDFLAGS= cmake \
+    LDFLAGS="-L/usr/local/opt/openssl/lib"
+    CXXFLAGS="-I/usr/local/opt/openssl/include"
+
+    run env CFLAGS="$CFLAGS" CXXFLAGS= LDFLAGS= cmake \
```

Comment:
```suggestion
    run env CFLAGS="$CFLAGS" CXXFLAGS="$CXXFLAGS" LDFLAGS="$LDFLAGS" cmake \
```

Reply:
I would change the `run env` below too. GitHub won't let me add a suggestion there.